engine_comparison/begin_experiment.sh

```diff
@@ -7,46 +7,58 @@
 DD=$(date +%d)
 MM=$(date +%m)
 INSTANCE_NAME="dispatcher-${DD}-${MM}"
-DISPATCHER_IMAGE_FAMILY=${DISPATCHER_IMAGE_FAMILY:-"ubuntu-1604-lts"} # Maybe container optimized
-export PROJECT_NAME="google.com:fuzz-comparisons"
+# DISPATCHER_IMAGE_FAMILY=${DISPATCHER_IMAGE_FAMILY:-"ubuntu-1604-lts"} # Maybe container optimized
+# export PROJECT_NAME="google.com:fuzz-comparisons"
 
 # These will frequently/usually be defined by the user
 export JOBS=${JOBS:-8}
 export N_ITERATIONS=${N_ITERATIONS:-5}
 
 # Define $BENCHMARKS
 if [[ $1 == 'all' ]]; then
-  for b in $(find ${SCRIPT_DIR}/*/build.sh -type f); do
+  for b in $(find ${SCRIPT_DIR}/../*/build.sh -type f); do
```

Comment:
This looks suspicious.  `SCRIPT_DIR` should be wherever `common.sh` is located.  So this is searching both the fuzzer-test-suite directory AND any sibling directories.